Makefile

```
@@ -90,28 +105,32 @@ else
 endif
 
 template_init:
-	@echo "\e[91m\nWarning! This will clean your template. Do you want to continue? [y/n]\e[0m" && read answer
-	if [ "$$answer" != "y" ]; then echo "Aborting!" && exit; fi
-
-	@echo "\nStarting to clean your template!\n\e[0m"
-
-	for folder in "template_example" "example/vagrant_box_example" "CHANGELOG.md"
-	do
-		echo -n "Deleting: $$folder "
-		rm -rf $$folder && echo "\e[32mSuccess\e[0m" || echo "\e[91mFailed\e[0m"
+	# 
+	@echo "${RED}\nWarning! This will clean your template. Do you want to continue? [y/n]${RESET}" ; \
+	read answer; \
+	if [ "$$answer" != "y" ]; then \
+		echo "Aborting!" ; \
+		exit 1 ; \
+	fi 
+
+	@echo "\nStarting to clean your template!" 
+
+	@for folder in "template_example" "example/vagrant_box_example" "CHANGELOG.md" ; do \
+		echo "Deleting: $$folder " ; \
+		rm -rf $$folder && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}" ; \
 	done
 
-	@echo -n "\nMoving README.md to .github/template_specific as old_README.md "
-	mv README.md .github/template_specific/old_README.md && echo "\e[32mSuccess\e[0m" || echo "\e[91mFailed\e[0m"
+	@echo "\nMoving README.md to .github/template_specific as old_README.md" 
+	@mv README.md .github/template_specific/old_README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
```

I think these should be printed to terminal, because they have no clear output from the cmd itself
```suggestion
	mv README.md .github/template_specific/old_README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
```